packages/truffle-contract/package.json

```
@@ -4,7 +4,7 @@
   "description": "A better contract abstraction for Ethereum (formerly EtherPudding)",
   "main": "index.js",
   "scripts": {
-    "prepare": "npm run compile",
+    "prepare": "yarn compile",
     "publish:next": "node ../truffle/scripts/prereleaseVersion.js next next",
     "test": "./scripts/test.sh",
     "test:debug": "$(npm bin)/mocha --inspect-brk",
```

```suggestion
     "test:debug": "$(yarn bin)/mocha --inspect-brk",
```
Same for `"test:trace"` below.